apps/application/serializers/application_serializers.py

```diff
@@ -684,7 +684,10 @@ def import_(self, with_valid=True):
                 self.is_valid()
             user_id = self.data.get('user_id')
             mk_instance_bytes = self.data.get('file').read()
-            mk_instance = pickle.loads(mk_instance_bytes)
+            try:
+                mk_instance = pickle.loads(mk_instance_bytes)
+            except Exception as e:
+                raise AppApiException(1001, "不支持的文件格式")
             application = mk_instance.application
             function_lib_list = mk_instance.function_lib_list
             if len(function_lib_list) > 0:
```

Comment:
The code itself is mostly well-structured and correct, but there are a couple of improvements that can be made:

1. **Exception Message**: It would be more informative to include additional context in the exception message when loading the pickle file fails, such as the type of error encountered.

```python
try:
    mk_instance = pickle.loads(mk_instance_bytes)
except pickle.UnpicklingError as e:
    # Improve Exception message by providing more information
    raise AppApiException(1001, f"无法加载pickle文件，错误消息: {e}")
```

2. **Code Readability**: Consider breaking down the import statement into multiple lines if it becomes too long:

```python
# Avoid overly long lines
mk_instance_bytes = (
    self.data.get('file')
)

# Continue the process after reading the bytes
```

These optimizations enhance the readability and maintainability of the code while also improving error handling. Make sure the updated exceptions contain helpful diagnostic information or relevant error codes so developers can easily diagnose issues like unsupported file formats during debugging.